### PR TITLE
FRW-5799 Added  Install the Data Exchange API + Inventory Management feature

### PR DESCRIPTION
**Pull request by dimitriyTsemma**
## PR Description

Ticket: https://spryker.atlassian.net/browse/FRW-5809

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        return [
+            new AvailabilityDynamicEntityPostCreatePlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that after updating stock data - product availability is updated as well.
```
Comment (asmarovydlo) [inline docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md:69]: Please provide an exact example of how I can do this. Data Exchange API request to update stock, + request to validate that availability is updated.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+Content-Length: 174
+{
+    "data": [
+        {
+            "idStockProduct": 1,
+            "Quantity": 10
```
Comment (asmarovydlo) [inline docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md:98]:
```suggestion
            "quantity": 10
```

File: docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+```bash
+POST /token/ HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/x-www-form-urlencoded
+Accept: application/json
+Content-Length: 67
+
+grant_type=password&username={username}&password={password}
+```
```
Comment (asmarovydlo) [inline docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md:82]: Please mention this link instead https://docs.spryker.com/docs/scos/dev/glue-api-guides/202307.0/dynamic-data-api/how-to-guides/how-to-send-request-in-data-exchange-api.html

File: docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        return [
+            new AvailabilityDynamicEntityPostCreatePlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Let’s say you have an endpoint `/dynamic-data/stock-products` to operate with data in `spy_stock_product` table in database.
```
Comment (asmarovydlo) [inline docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md:69]: Please mention which configuration should be added to DB to allow this request.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

**Review by asmarovydlo — DISMISSED** [review]

File: docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+COMMIT;
+```
+
+To obtain an access token follow [How to send a request in Data Exchange API](/docs/scos/dev/glue-api-guides/{{page.version}}/dynamic-data-api/how-to-guides/how-to-send-request-in-data-exchange-api.html)
+
+### Sending a `PATCH` request
+This request needs to include the necessary headers, such as Content-Type, Accept, and Authorization, with the access token provided.
+
```
Comment (lenadoc) [inline docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md:83]: Can we remove some of the information and instead give reference to the howto guide? That is, I would suggest replacing lines 69-82 with the following text:
 "Let's say you want to have a new endpoint `/dynamic-data/stock-products` to operate with data in `spy_stock_product` table in database. Send the PATCH request to update the product stock data according to the guidelines in [How to send a request in Data Exchange] API (/docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-send-request-in-data-exchange-api.html):"
 
 What do you think?

Comment (dimitriyTsemma) [reply docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/install-the-data-exchange-api-inventory-management-feature.md:83]: we don`t have information about the `stock-products` endpoint in the How To guide, so it looks like we can`t remove anything here (

**Pull request merged**